gemlite/triton_kernels/gemv_A16fWnO16f_int32packing.py

```diff
@@ -51,9 +51,14 @@ def get_gemv_config():
 
     return _configs
 
+def get_closest_m(M):
```

Comment:
You don't need to do it in the gemv, since it will only run for batch-sizes <=8

Reply:
I see, so you suggest tuning for each batch size from 1 to 8 is fine? I guess that should be fine, I just limited it to 4 values -> 1,2,4,8 instead of 8 to make the total tuning time a bit faster. So for example, if input batch size is 5 it will consider `CLOSEST_M` as 4 and next time an input with batch size 4,5 or 6 comes it won't tune again since all will be have the same `CLOSEST_M=4`.